lib/rules/object-curly-spacing.js

```diff
@@ -32,6 +32,9 @@ module.exports = {
                     },
                     objectsInObjects: {
                         type: "boolean"
+                    },
+                    spaceInEmptyObject: {
+                        type: "string"
```

Comment:
I think this should be an `enum` so we can enforce always/never specifically. (See line 25 for another example)